dotnet/README.md

```diff
@@ -2,13 +2,13 @@
 
 ## .NET environment setup
 
-You can skip this part if you already have the .NET Core 3.x framework installed.
+You can skip this part if you already have the .NET Core 7.x framework installed.
 
-### .NET Core SDK 3.x
+### .NET Core SDK 7.x
```

Comment:
It's not named "Core" after v3.1: they simplified and jumped to .NET 5: https://learn.microsoft.com/en-us/dotnet/core/introduction#net-history